R/RunDiagnostics.R

```diff
@@ -687,6 +692,37 @@ executeDiagnostics <- function(cohortDefinitionSet,
       }
     )
   }
+  
+  # Defines variables and checks version of external concept counts table -----
+  if (!useExternalConceptCountsTable) {
+    conceptCountsTableIsTemp <- TRUE
+    if (conceptCountsTable != "#concept_counts") {
+      conceptCountsTable <- "#concept_counts"
+    }
+  } else {
+    if (conceptCountsTable == "#concept_counts") {
```

Comment:
This should probably be ```substr(conceptCountsTable, 1, 1) == "#"``` as otherwise `#my_temp_table` will render as `work_database_schema.#my_temp_table` which will crash on some systems